### PR TITLE
zktree:: init at 0.0.1

### PR DESCRIPTION
**Pull request by alirezameskin**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [X] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/zktree/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    };
+
+    cargoSha256 = "1d35jrxvhf7m04s1kh0yrfhy9j9i6qzwbw2mwapgsrcsr5vhxasn";
+
+    meta = with lib; {
+            description = "A small tool to display Znodes in Zookeeper in tree structure.";
+            homepage = https://github.com/alirezameskin/zktree;
+            license = licenses.unlicense;
+            maintainers = with lib.maintainers; [ alirezameskin ];
+            platforms = platforms.all;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/zktree/default.nix:25]:
```suggestion
```

File: pkgs/applications/misc/zktree/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        repo = "zktree";
+        rev = version;
+        sha256 = "11w86k1w5zryiq6bqr98pjhffd3l76377yz53qx0n76vc5374fk9";
+    };
+
+    cargoSha256 = "1d35jrxvhf7m04s1kh0yrfhy9j9i6qzwbw2mwapgsrcsr5vhxasn";
+
+    meta = with lib; {
+            description = "A small tool to display Znodes in Zookeeper in tree structure.";
+            homepage = https://github.com/alirezameskin/zktree;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/zktree/default.nix:22]:
```suggestion
            homepage = "https://github.com/alirezameskin/zktree";
```

**Comment by SuperSandro2000** [comment]
Can you format your changes with an indentation of 2 tabs?

**Comment by alirezameskin** [comment]
@SuperSandro2000 done

File: pkgs/applications/misc/zktree/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{ callPackage
+, lib
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/zktree/default.nix:2]:
```suggestion
{ lib
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Can you squash the review commits together?

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101468` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>zktree</li>
  </ul>
</details>

File: maintainers/maintainer-list.nix [hunk 4]
```diff
@@ -378,6 +378,12 @@
     githubId = 2822871;
     name = "Alistair Bill";
   };
+  alirezameskin = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:381]: Please put this into a separate commit with the message ``maintainers: add alirezameskin``.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101468` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>zktree</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101468` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>zktree</li>
  </ul>
</details>
